<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcome_label">
     <property name="text">
      <string>Добро пожаловать в игру крестики нолики на Qt!</string>
     </property>
     <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_field_size">
     <item>
      <widget class="QLabel" name="label_field_size">
       <property name="text">
        <string>Размер поля</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="field_size">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_field_size">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_first_player">
     <item>
      <widget class="QLabel" name="label_first_player">
       <property name="text">
        <string>Имя первого игрока</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="first_player"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_second_player">
     <item>
      <widget class="QLabel" name="label_second_player">
       <property name="text">
        <string>Имя второго игрока</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="second_player"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>field_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_field_size</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_field_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>field_size</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
